src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+}
+
+.search_form {
```

Comment:
<img width="900" alt="Screenshot 2023-10-03 at 21 45 29" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/83630443/7af928c1-bac7-429b-843e-8f6d29e27bb3">
follow this naming ! [source](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#CSS)